build/tasks/aria-supported.js

```diff
@@ -60,6 +61,33 @@ module.exports = function(grunt) {
 									!subject.hasOwnProperty(key)
 								) {
 									out.push([`${key}`, 'No']);
+								} else if (
```

Comment:
the complexity of this `reduce` fn has become hard to follow, worth splitting this into smaller fn's, with detailed comments.